api/client/client.go

```diff
@@ -45,11 +44,20 @@ type Remote interface {
 func NewServer(addr string) Remote {
 	addrs := strings.Split(addr, ",")
 
+	for i := range addrs {
```

Comment:
Good point. Maybe check if scheme ends up being http or https, and then append http:// if it is not and parse again? In any case, this normalization logic should be in one place.